dockerfiles/alpine/micro-integrator/Dockerfile

```diff
@@ -54,9 +55,12 @@ RUN \
 COPY --chown=wso2carbon:wso2 docker-entrypoint.sh ${USER_HOME}/
 # install required packages
 RUN apk add --no-cache netcat-openbsd
+# add CA cerficates
```

Comment:
@NishikaDeSilva @ThilinaManamgoda can't we combine the `apk add` package addition command into a single instruction?

Reply:
Changes made